content/docs/1.6.0/best-practices.md

```diff
@@ -99,6 +99,46 @@ Any extra disks must be written in the `/etc/fstab` file to allow automatic moun
 
 Don't use a symbolic link for the extra disks. Use `mount --bind` instead of `ln -s` and make sure it's in the `fstab` file. For details, see [the section about multiple disk support.](../volumes-and-nodes/multidisk/#use-an-alternative-path-for-a-disk-on-the-node)
 
+### Disk Performance Optimization  
```

Comment:
Suggest moving this section to H1, and changing the title to `Volume Performance Optimization`. Add this section before `Deploying Workload`.

Also, remember to update the TOC.